userspace/falco/falco_metrics.cpp

```diff
@@ -72,9 +72,22 @@ std::string falco_metrics::to_text(const falco::app::state& state) {
 	libs::metrics::prometheus_metrics_converter prometheus_metrics_converter;
 	std::string prometheus_text;
 
-	for(auto inspector : inspectors) {
-		// Falco wrapper metrics
+	for(size_t i = 0; i < inspectors.size(); ++i) {  // Start inspector loop
+		auto& inspector = inspectors[i];
+
+		// Falco wrapper metrics, repeated for each inspector, accounting for plugins w/ event
+		// sources
 		//
+
+		/* Examples ...
```

Comment:
@sgaist this seems to be the only true reason for why we would need to loop over each inspector vs only needing the first one.  The examples listed here were verified by me when testing with the dummy_c event source plugin.